src/test/java/co/rsk/federate/BtcToRskClientTest.java

```diff
@@ -2432,6 +2465,40 @@ void updateBridge_when_does_not_hasBetterBlockToSync_updates_headers_coinbase_tr
         verify(federatorSupport, times(1)).sendUpdateCollections();
     }
 
+    @Test
+    void updateBridge_whenUpdateBridgeConfigAreFalse_shouldNotCallAny() throws Exception {
```

Comment:
Could you add one more test where the config settings are not defined in the file? According to your change if that's the case the services should run well.

Thanks

Reply:
Updated!